main.go

```diff
@@ -6,7 +6,7 @@ import (
 	"path"
 	"strconv"
 
-	"github.com/codegangsta/cli"
+	"github.com/docker/machine/cli"
```

Comment:
I could definitely do that, and arguably that'd be the right way to handle it.  If the PR I mentioned above doesn't get merged in, that's likely what I'll end up doing.